build.gradle

```diff
@@ -0,0 +1,60 @@
+plugins {
+  id "de.undercouch.download" version "4.1.1"
+}
+
+apply plugin: 'java'
+apply plugin: 'de.undercouch.download'
+
+group = 'org.lfenergy.compas'
+version = '0.1-SNAPSHOT'
+
+description = """"""
+
+import de.undercouch.gradle.tasks.download.Download
+import de.undercouch.gradle.tasks.download.Verify
+
+task downloadMultipleFiles(type: Download) {
+  src([
+    'https://assets.iec.ch/public/tc57/IEC_61850-6.2003.SCL.1.7.full.zip',
+    'https://assets.iec.ch/public/tc57/IEC_61850-6.2009.SCL.2007B.full.zip',
+    'https://assets.iec.ch/public/tc57/IEC_61850-6.2018.SCL.2007B4.full.zip'
+  ])
+  dest project.buildDir
+}
+
+task verify1_7ZipFile(dependsOn: downloadMultipleFiles, type: Verify) {
```

Comment:
I think it's better to create separate gradle files for each type of XSDs (1.7, 2007B and 2007B4) for better maintenance.